[WASM] Wasm.Build.Tests run with `EMSDK` - `clang` failing to load `libLLVM-13git.so`

Configuration: `net7.0-Browser-Release-wasm-Mono_Release_Linux-BuildWasmApps-Ubuntu.1804.Amd64.Open`
CI build: [runtime pipeline of the rolling build](https://dev.azure.com/dnceng/public/_build/results?buildId=1635569&view=ms.vss-test-web.build-test-results-tab&runId=45291016&paneView=debug&resultId=100051)

An example of failed tests:
```
      <test name="Wasm.Build.NativeRebuild.Tests.NoopNativeRebuildTest.NoOpRebuildForNativeBuilds(buildArgs: BuildArgs { ProjectName = placeholder, Config = Release, AOT = False, ProjectFileContents = placeholder, ExtraBuildArgs =  }, nativeRelink: True, invariant: False, host: V8, id: \&quot;h420la1w.j0k\&quot;)" type="Wasm.Build.NativeRebuild.Tests.NoopNativeRebuildTest" method="NoOpRebuildForNativeBuilds" time="54.2599321" result="Fail">
        <output><![CDATA[-------- Building ---------
Binlog path: /datadisks/disk1/work/A0100914/w/B4AD0967/uploads/xharness-output/logs/h420la1w.j0k/rebuild_noop_Release.binlog
Running /datadisks/disk1/work/A0100914/p/sdk-no-workload/dotnet publish -p:WasmBuildOnlyAfterPublish=true  /p:WasmBuildSupportDir=/datadisks/disk1/work/A0100914/p/build /p:EMSDK_PATH=/datadisks/disk1/work/A0100914/p/build/emsdk  /p:Configuration=Release /bl:"/datadisks/disk1/work/A0100914/w/B4AD0967/uploads/xharness-output/logs/h420la1w.j0k/rebuild_noop_Release.binlog" /nologo /fl /flp:"v:diag,LogFile=/datadisks/disk1/work/A0100914/w/B4AD0967/uploads/xharness-output/logs/h420la1w.j0k/rebuild_noop_Release.binlog.log" /v:minimal  
WorkingDirectory: /datadisks/disk1/work/A0100914/w/B4AD0967/e/h420la1w.j0k
Setting environment variables for execution:
	EMSDK_PATH = /datadisks/disk1/work/A0100914/p/build/emsdk
	DOTNET_INSTALL_DIR = /datadisks/disk1/work/A0100914/p/sdk-no-workload
	DOTNET_MULTILEVEL_LOOKUP = 0
	DOTNET_SKIP_FIRST_TIME_EXPERIENCE = 1
	MSBuildSDKsPath = 
	PATH = /datadisks/disk1/work/A0100914/p/sdk-no-workload:/datadisks/disk1/work/A0100914/p/sdk-no-workload:/datadisks/disk1/work/A0100914/p/xharness-cli:/datadisks/disk1/work/A0100914/p/dotnet-cli:/datadisks/disk1/work/A0100914/p/build/emsdk/node/14.15.5_64bit/bin:/datadisks/disk1/work/A0100914/p/chrome-linux:/datadisks/disk1/work/A0100914/p/chromedriver_linux64:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
	WasmNativeStrip = false
[h420la1w.j0k] 
[h420la1w.j0k] Welcome to .NET 7.0!
[h420la1w.j0k] ---------------------
[h420la1w.j0k] SDK Version: 7.0.100-alpha.1.21528.1
[h420la1w.j0k] 
[h420la1w.j0k] ----------------
[h420la1w.j0k] Installed an ASP.NET Core HTTPS development certificate.
[h420la1w.j0k] To trust the certificate run 'dotnet dev-certs https --trust' (Windows and macOS only).
[h420la1w.j0k] Learn about HTTPS: https://aka.ms/dotnet-https
[h420la1w.j0k] ----------------
[h420la1w.j0k] Write your first app: https://aka.ms/dotnet-hello-world
[h420la1w.j0k] Find out what's new: https://aka.ms/dotnet-whats-new
[h420la1w.j0k] Explore documentation: https://aka.ms/dotnet-docs
[h420la1w.j0k] Report issues and find source on GitHub: https://github.com/dotnet/core
[h420la1w.j0k] Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cli
[h420la1w.j0k] --------------------------------------------------------------------------------------
[h420la1w.j0k] /datadisks/disk1/work/A0100914/p/sdk-no-workload/sdk/7.0.100-alpha.1.21528.1/MSBuild.dll /fl /nologo -maxcpucount -property:WasmBuildSupportDir=/datadisks/disk1/work/A0100914/p/build -property:EMSDK_PATH=/datadisks/disk1/work/A0100914/p/build/emsdk -property:Configuration=Release -p:WasmBuildOnlyAfterPublish=true -restore -target:Publish -verbosity:m /v:minimal /bl:/datadisks/disk1/work/A0100914/w/B4AD0967/uploads/xharness-output/logs/h420la1w.j0k/rebuild_noop_Release.binlog /flp:v:diag,LogFile=/datadisks/disk1/work/A0100914/w/B4AD0967/uploads/xharness-output/logs/h420la1w.j0k/rebuild_noop_Release.binlog.log ./rebuild_noop_Release.csproj
[h420la1w.j0k]   Determining projects to restore...
[h420la1w.j0k]   Restored /datadisks/disk1/work/A0100914/w/B4AD0967/e/h420la1w.j0k/rebuild_noop_Release.csproj (in 46.12 sec).
[h420la1w.j0k]   You are using a preview version of .NET. See: https://aka.ms/dotnet-core-preview
[h420la1w.j0k]   rebuild_noop_Release -> /datadisks/disk1/work/A0100914/w/B4AD0967/e/h420la1w.j0k/bin/Release/net6.0/browser-wasm/rebuild_noop_Release.dll
[h420la1w.j0k]   ** MicrosoftNetCoreAppRuntimePackDir : /datadisks/disk1/work/A0100914/p/build/microsoft.netcore.app.runtime.browser-wasm/
[h420la1w.j0k]   Optimizing assemblies for size, which may change the behavior of the app. Be sure to test after publishing. See: https://aka.ms/dotnet-illink
[h420la1w.j0k]   rebuild_noop_Release -> /datadisks/disk1/work/A0100914/w/B4AD0967/e/h420la1w.j0k/bin/Release/net6.0/browser-wasm/publish/
[h420la1w.j0k]   rebuild_noop_Release -> /datadisks/disk1/work/A0100914/w/B4AD0967/e/h420la1w.j0k/bin/Release/net6.0/browser-wasm/rebuild_noop_Release.dll
[h420la1w.j0k]   ** MicrosoftNetCoreAppRuntimePackDir : /datadisks/disk1/work/A0100914/p/build/microsoft.netcore.app.runtime.browser-wasm/
[h420la1w.j0k]   Optimizing assemblies for size, which may change the behavior of the app. Be sure to test after publishing. See: https://aka.ms/dotnet-illink
[h420la1w.j0k]   rebuild_noop_Release -> /datadisks/disk1/work/A0100914/w/B4AD0967/e/h420la1w.j0k/bin/Release/net6.0/browser-wasm/publish/
[h420la1w.j0k] embuilder : error : '/datadisks/disk1/work/A0100914/p/build/emsdk/upstream/bin/clang --version' failed (returned 127) [/datadisks/disk1/work/A0100914/w/B4AD0967/e/h420la1w.j0k/rebuild_noop_Release.csproj]
[h420la1w.j0k] /datadisks/disk1/work/A0100914/p/build/wasm/WasmApp.Native.targets(317,5): error MSB3073: The command "embuilder.py build MINIMAL" exited with code 1. [/datadisks/disk1/work/A0100914/w/B4AD0967/e/h420la1w.j0k/rebuild_noop_Release.csproj]
]]></output>
        <failure exception-type="Xunit.Sdk.TrueException">
          <message><![CDATA[Build process exited with non-zero exit code: 1\nExpected: True\nActual:   False]]></message>
          <stack-trace><![CDATA[   at Wasm.Build.Tests.BuildTestBase.AssertBuild(String args, String label, Boolean expectSuccess, IDictionary`2 envVars, Nullable`1 timeoutMs) in /_/src/tests/BuildWasmApps/Wasm.Build.Tests/BuildTestBase.cs:line 588
   at Wasm.Build.Tests.BuildTestBase.BuildProject(BuildArgs buildArgs, String id, BuildProjectOptions options) in /_/src/tests/BuildWasmApps/Wasm.Build.Tests/BuildTestBase.cs:line 337
   at Wasm.Build.NativeRebuild.Tests.NativeRebuildTestsBase.FirstNativeBuild(String programText, Boolean nativeRelink, Boolean invariant, BuildArgs buildArgs, String id, String extraProperties) in /_/src/tests/BuildWasmApps/Wasm.Build.Tests/NativeRebuildTests/NativeRebuildTestsBase.cs:line 50
   at Wasm.Build.NativeRebuild.Tests.NoopNativeRebuildTest.NoOpRebuildForNativeBuilds(BuildArgs buildArgs, Boolean nativeRelink, Boolean invariant, RunHost host, String id) in /_/src/tests/BuildWasmApps/Wasm.Build.Tests/NativeRebuildTests/NoopNativeRebuildTest.cs:line 26]]></stack-trace>
        </failure>
```

These tests are failing when running WBT with emsdk from the repo, instead of workloads.

Looking at the binlog:
```
                   Task "Exec" (TaskId:231)
                     Task Parameter:
                         EnvironmentVariables=
                             DOTNET_EMSCRIPTEN_LLVM_ROOT=/datadisks/disk1/work/A0100914/p/build/emsdk/bin
                             DOTNET_EMSCRIPTEN_BINARYEN_ROOT=/datadisks/disk1/work/A0100914/p/build/emsdk/
                             DOTNET_EMSCRIPTEN_NODE_JS=/datadisks/disk1/work/A0100914/p/build/emsdk/node/14.15.5_64bit/bin/node
                             PATH=/datadisks/disk1/work/A0100914/p/build/emsdk:/datadisks/disk1/work/A0100914/p/build/emsdk/upstream/bin:/datadisks/disk1/work/A0100914/p/build/emsdk/upstream/emscripten:/datadisks/disk1/work/A0100914/p/build/emsdk/node/14.15.5_64bit/bin:/datadisks/disk1/work/A01009
                             PYTHONUTF8=1 (TaskId:231)
                     Task Parameter:StandardErrorImportance=Low (TaskId:231)
                     Task Parameter:StandardOutputImportance=Low (TaskId:231)
                     Task Parameter:Command=embuilder.py build MINIMAL (TaskId:231)
                     Environment Variables passed to tool: (TaskId:231)
                       DOTNET_EMSCRIPTEN_LLVM_ROOT=/datadisks/disk1/work/A0100914/p/build/emsdk/bin (TaskId:231)
                       DOTNET_EMSCRIPTEN_BINARYEN_ROOT=/datadisks/disk1/work/A0100914/p/build/emsdk/ (TaskId:231)
                       DOTNET_EMSCRIPTEN_NODE_JS=/datadisks/disk1/work/A0100914/p/build/emsdk/node/14.15.5_64bit/bin/node (TaskId:231)
                       PATH=/datadisks/disk1/work/A0100914/p/build/emsdk:/datadisks/disk1/work/A0100914/p/build/emsdk/upstream/bin:/datadisks/disk1/work/A0100914/p/build/emsdk/upstream/emscripten:/datadisks/disk1/work/A0100914/p/build/emsdk/node/14.15.5_64bit/bin:/datadisks/disk1/work/A0100914/p/s
                       PYTHONUTF8=1 (TaskId:231)
                     embuilder.py build MINIMAL (TaskId:231)
                     /datadisks/disk1/work/A0100914/p/build/emsdk/upstream/bin/clang: error while loading shared libraries: libLLVM-13git.so: cannot open shared object file: No such file or directory (TaskId:231)
08:55:48.036   1:8>embuilder : error : '/datadisks/disk1/work/A0100914/p/build/emsdk/upstream/bin/clang --version' failed (returned 127) [/datadisks/disk1/work/A0100914/w/AD9D09CE/e/f323ik3b.3wg/build_publish_Release.csproj]
08:55:48.036   1:8>/datadisks/disk1/work/A0100914/p/build/wasm/WasmApp.Native.targets(317,5): error MSB3073: The command "embuilder.py build MINIMAL" exited with code 1. [/datadisks/disk1/work/A0100914/w/AD9D09CE/e/f323ik3b.3wg/build_publish_Release.csproj]
```

Specifically:
`/datadisks/disk1/work/A0100914/p/build/emsdk/upstream/bin/clang: error while loading shared libraries: libLLVM-13git.so: cannot open shared object file: No such file or directory (TaskId:231)`

cc @radekdoulik

--- a/src/libraries/sendtohelix-wasm.targets
+++ b/src/libraries/sendtohelix-wasm.targets
@@ -20,13 +20,22 @@
     <EmSdkDirForHelixPayload>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'src', 'mono', 'wasm', 'emsdk'))</EmSdkDirForHelixPayload>
 
     <NeedsWorkload Condition="'$(Scenario)' == 'BuildWasmApps'">true</NeedsWorkload>
-    <NeedsEMSDK Condition="'$(NeedsToBuildWasmAppsOnHelix)' == 'true' or ('$(Scenario)' == 'BuildWasmApps' and '$(TestUsingWorkloads)' != 'true')">true</NeedsEMSDK>
+    <NeedsEMSDK Condition="'$(NeedsToBuildWasmAppsOnHelix)' == 'true' or '$(Scenario)' == 'BuildWasmApps'">true</NeedsEMSDK>
     <NeedsEMSDKNode Condition="'$(Scenario)' == 'WasmTestOnNodeJs' or '$(Scenario)' == 'BuildWasmApps'">true</NeedsEMSDKNode>
     <NeedsToRunOnBrowser Condition="'$(Scenario)' == 'WasmTestOnBrowser' or '$(Scenario)' == 'BuildWasmApps'">true</NeedsToRunOnBrowser>
     <NeedsToRunOnBrowser Condition="'$(NeedsToRunOnBrowser)' == '' and '$(IsWasmDebuggerTests)' == 'true'">true</NeedsToRunOnBrowser>
 
     <IncludeXHarnessCli>true</IncludeXHarnessCli>
     <EnableXHarnessTelemetry>true</EnableXHarnessTelemetry>
+    <IncludeNodePayload Condition="'$(NeedsEMSDKNode)' == 'true' and '$(NeedsEMSDK)' != 'true'">true</IncludeNodePayload>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <_HelixLocalNodePath Condition="'$(NeedsEMSDK)' == 'true' and '$(WindowsShell)' != 'true'">$HELIX_CORRELATION_PAYLOAD/build/emsdk/node</_HelixLocalNodePath>
+    <_HelixLocalNodePath Condition="'$(NeedsEMSDK)' == 'true' and '$(WindowsShell)' == 'true'">%HELIX_CORRELATION_PAYLOAD%\build\emsdk\node</_HelixLocalNodePath>
+
+    <_HelixLocalNodePath Condition="'$(NeedsEMSDKNode)' == 'true' and '$(WindowsShell)' != 'true'">$HELIX_CORRELATION_PAYLOAD/build/emsdk-node</_HelixLocalNodePath>
+    <_HelixLocalNodePath Condition="'$(NeedsEMSDKNode)' == 'true' and '$(WindowsShell)' == 'true'">%HELIX_CORRELATION_PAYLOAD%\build\emsdk-node</_HelixLocalNodePath>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(WindowsShell)' != 'true'">
@@ -40,15 +49,6 @@
 
     <HelixPreCommand Condition="'$(NeedsToRunOnBrowser)' == 'true'" Include="export PATH=$HELIX_CORRELATION_PAYLOAD/$(ChromeDriverDirName):$PATH" />
     <HelixPreCommand Condition="'$(NeedsToRunOnBrowser)' == 'true'" Include="export PATH=$HELIX_CORRELATION_PAYLOAD/$(ChromiumDirName):$PATH" />
-
-    <!-- Fix symbolic links that are broken already on build machine and also in the correlation payload -->
-    <HelixPreCommand Condition="'$(NeedsEMSDKNode)' == 'true'" Include="export HELIX_NODEJS_VERSION=%24(ls $HELIX_CORRELATION_PAYLOAD/build/emsdk/node)" />
-    <HelixPreCommand Condition="'$(NeedsEMSDKNode)' == 'true'" Include="export HELIX_NODEJS_PATH=$HELIX_CORRELATION_PAYLOAD/build/emsdk/node/$HELIX_NODEJS_VERSION" />
-    <HelixPreCommand Condition="'$(NeedsEMSDKNode)' == 'true'" Include="export PATH=$HELIX_NODEJS_PATH/bin:$PATH" />
-    <HelixPreCommand Condition="'$(NeedsEMSDKNode)' == 'true'" Include="rm $HELIX_NODEJS_PATH/bin/npm" />
-    <HelixPreCommand Condition="'$(NeedsEMSDKNode)' == 'true'" Include="rm $HELIX_NODEJS_PATH/bin/npx" />
-    <HelixPreCommand Condition="'$(NeedsEMSDKNode)' == 'true'" Include="ln -s ../lib/node_modules/npm/bin/npm-cli.js $HELIX_NODEJS_PATH/bin/npm" />
-    <HelixPreCommand Condition="'$(NeedsEMSDKNode)' == 'true'" Include="ln -s ../lib/node_modules/npm/bin/npx-cli.js $HELIX_NODEJS_PATH/bin/npx" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(WindowsShell)' == 'true'">
@@ -62,12 +62,26 @@
 
     <HelixPreCommand Condition="'$(NeedsToRunOnBrowser)' == 'true'" Include="set PATH=%HELIX_CORRELATION_PAYLOAD%\$(ChromeDriverDirName)%3B%PATH%" />
     <HelixPreCommand Condition="'$(NeedsToRunOnBrowser)' == 'true'" Include="set PATH=%HELIX_CORRELATION_PAYLOAD%\$(ChromiumDirName)%3B%PATH%" />
+  </ItemGroup>
 
-    <HelixPreCommand Condition="'$(NeedsEMSDKNode)' == 'true'" Include="for /f %%i in ('dir %HELIX_CORRELATION_PAYLOAD%\build\emsdk\node /b') do set HELIX_NODEJS_VERSION=%%i" />
-    <HelixPreCommand Condition="'$(NeedsEMSDKNode)' == 'true'" Include="set PATH=%HELIX_CORRELATION_PAYLOAD%\build\emsdk\node\%HELIX_NODEJS_VERSION%\bin%3B%PATH%" />
+  <ItemGroup Condition="'$(NeedsEMSDKNode)' == 'true' and '$(WindowsShell)' != 'true'">
+    <!-- Fix symbolic links that are broken already on build machine and also in the correlation payload -->
+    <HelixPreCommand Include="export _HELIX_NODEJS_VERSION=%24(ls $(_HelixLocalNodePath))" />
+    <HelixPreCommand Include="export _HELIX_NODEJS_PATH=$(_HelixLocalNodePath)/$_HELIX_NODEJS_VERSION" />
+    <HelixPreCommand Include="export PATH=$_HELIX_NODEJS_PATH/bin:$PATH" />
+    <HelixPreCommand Include="rm $_HELIX_NODEJS_PATH/bin/npm" />
+    <HelixPreCommand Include="rm $_HELIX_NODEJS_PATH/bin/npx" />
+    <HelixPreCommand Include="ln -s ../lib/node_modules/npm/bin/npm-cli.js $_HELIX_NODEJS_PATH/bin/npm" />
+    <HelixPreCommand Include="ln -s ../lib/node_modules/npm/bin/npx-cli.js $_HELIX_NODEJS_PATH/bin/npx" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(NeedsEMSDKNode)' == 'true' and '$(WindowsShell)' == 'true'">
+    <HelixPreCommand Include="for /f %%i in ('dir $(_HelixLocalNodePath) /b') do set _HELIX_NODEJS_VERSION=%%i" />
+    <HelixPreCommand Include="set PATH=$(_HelixLocalNodePath)/%_HELIX_NODEJS_VERSION%/bin%3B%PATH%" />
   </ItemGroup>
 
   <PropertyGroup>
+
     <!--
       We are hosting the payloads for the WASM/browser on kestrel in the xharness process.
       We also run some network tests to this server and so, we are running it on both HTTP and HTTPS.
@@ -123,8 +137,9 @@
       <HelixCorrelationPayload Include="$(MonoTargetsTasksDir)"                  Destination="build/MonoTargetsTasks" />
     </ItemGroup>
 
-    <ItemGroup Condition="'$(NeedsEMSDKNode)' == 'true'">
-      <HelixCorrelationPayload Include="$(EmSdkDirForHelixPayload)node"          Destination="build/emsdk/node" />
+    <!-- copy node separately only if EMSDK is not being included -->
+    <ItemGroup Condition="'$(IncludeNodePayload)' == 'true'">
+      <HelixCorrelationPayload Include="$(EmSdkDirForHelixPayload)node"          Destination="build/emsdk-node" />
     </ItemGroup>
 
     <ItemGroup Condition="'$(Scenario)' == '' or '$(Scenario)' == 'normal' or '$(Scenario)' == 'WasmTestOnBrowser' or '$(Scenario)' == 'WasmTestOnNodeJS'">
@@ -154,7 +169,7 @@
     </PropertyGroup>
 
     <Message Condition="'$(NeedsEMSDK)' == 'true' or '$(NeedsEMSDKNode)' == 'true'" Importance="High" Text="Using emsdk: $(EmSdkDirForHelixPayload)" />
-    
+
     <ItemGroup Condition="'$(IsRunningLibraryTests)' == 'true'">
       <_WasmWorkItem Include="$(TestArchiveRoot)browseronly/**/*.zip"                          Condition="'$(Scenario)' == 'WasmTestOnBrowser'" />
       <_WasmWorkItem Include="$(TestArchiveRoot)browserornodejs/**/*.zip"                      Condition="'$(Scenario)' == 'WasmTestOnBrowser'" />
@@ -208,7 +223,7 @@
   <!-- CI has emscripten provisioned in $(EMSDK_PATH) as `/usr/local/emscripten`. Because helix tasks will
        attempt to write a .payload file, we cannot use $(EMSDK_PATH) to package emsdk as a helix correlation
        payload. Instead, we copy over the files to a new directory `src/mono/wasm/emsdk` and use that. -->
-  <Target Name="StageEmSdkForHelix" Condition="('$(NeedsEMSDK)' == 'true' or '$(NeedsEMSDKNode)' == 'true') and !Exists($(EmSdkDirForHelixPayload))">
+  <Target Name="StageEmSdkForHelix" Condition="('$(NeedsEMSDK)' == 'true' or '$(IncludeNodePayload)' == 'true') and !Exists($(EmSdkDirForHelixPayload))">
     <Error Condition="'$(EMSDK_PATH)' == '' or !Exists($(EMSDK_PATH))"
            Text="Could not find emscripten sdk in EMSDK_PATH=$(EMSDK_PATH), needed to provision for running tests on helix" />
 

--- a/src/libraries/sendtohelix-wasm.targets
+++ b/src/libraries/sendtohelix-wasm.targets
@@ -31,11 +31,11 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <_HelixLocalNodePath Condition="'$(NeedsEMSDK)' == 'true' and '$(WindowsShell)' != 'true'">$HELIX_CORRELATION_PAYLOAD/build/emsdk/node</_HelixLocalNodePath>
-    <_HelixLocalNodePath Condition="'$(NeedsEMSDK)' == 'true' and '$(WindowsShell)' == 'true'">%HELIX_CORRELATION_PAYLOAD%\build\emsdk\node</_HelixLocalNodePath>
-
     <_HelixLocalNodePath Condition="'$(NeedsEMSDKNode)' == 'true' and '$(WindowsShell)' != 'true'">$HELIX_CORRELATION_PAYLOAD/build/emsdk-node</_HelixLocalNodePath>
     <_HelixLocalNodePath Condition="'$(NeedsEMSDKNode)' == 'true' and '$(WindowsShell)' == 'true'">%HELIX_CORRELATION_PAYLOAD%\build\emsdk-node</_HelixLocalNodePath>
+
+    <_HelixLocalNodePath Condition="'$(NeedsEMSDK)' == 'true' and '$(WindowsShell)' != 'true'">$HELIX_CORRELATION_PAYLOAD/build/emsdk/node</_HelixLocalNodePath>
+    <_HelixLocalNodePath Condition="'$(NeedsEMSDK)' == 'true' and '$(WindowsShell)' == 'true'">%HELIX_CORRELATION_PAYLOAD%\build\emsdk\node</_HelixLocalNodePath>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(WindowsShell)' != 'true'">

